src/foam/dao/journal/EasyJournalEntryDAO.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.journal',
+  name: 'EasyJournalEntryDAO',
```

Comment:
What is this for?

Reply:
Convenience for when we want to add more features to these journal entries. Helps ensure we use the right order when putting the decorators together.